.github/workflows/scheduled.yml

```diff
@@ -0,0 +1,128 @@
+name: Scheduled
+
+on:
+  pull_request:
+    paths:
+      - scripts/get_scarb_versions.sh
+      - .github/workflows/scheduled.yml
+  schedule:
+    - cron: '0 0 * * 3,6'
+
+jobs:
+  get-scarb-versions:
+    name: Get Scarb versions
+    outputs:
+      versions: ${{ steps.get_versions.outputs.versions }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: asdf-vm/actions/install@05e0d2ed97b598bfce82fd30daf324ae0c4570e6
+        with:
+          tool_versions: |
+            scarb latest
+
```

Comment:
What's the purpose of this?

Reply:
To execute commands like `asdf list all scarb` the Scarb plugin must first be installed. I expected that the asdf action would automatically install Scarb since it is listed in the `.tool_versions` file in the repo, but for some  reason this was not the case and it was necessary to explicitly provide this argument here. So the purpose is solely to install Scarb asdf plugin.